SampleApps/CarthageTest/CarthageTest/ViewController.swift

```diff
@@ -13,14 +13,18 @@ import BraintreeSEPADirectDebit
 class ViewController: UIViewController {
 
     override func viewDidLoad() {
-        super.viewDidLoad()
-    }
+        let apiClient = BTAPIClient(authorization: "sandbox_9dbg82cq_dcpspy2brwdjr3qn")!
 
-    override func didReceiveMemoryWarning() {
-        super.didReceiveMemoryWarning()
-        // Dispose of any resources that can be recreated.
+        let amexClient = BTAmericanExpressClient(apiClient: apiClient)
+        let applePayClient = BTApplePayClient(apiClient: apiClient)
+        let cardClient = BTCardClient(apiClient: apiClient)
+        let dataCollector = BTDataCollector(apiClient: apiClient)
+        let paymentFlowClient = BTPaymentFlowClient(apiClient: apiClient)
+        let payPalClient = BTPayPalClient(apiClient: apiClient)
+        let threeDSecureClient = BTThreeDSecureClient(apiClient: apiClient)
+        let venmoClient = BTVenmoClient(apiClient: apiClient)
+        let payPalNativeCheckoutClient = BTPayPalNativeCheckoutClient(apiClient: apiClient)
```

Comment:
Looks like we need to import the PayPalNativeCheckout module

Reply:
Yep - guess the other benefit of this is finding things we forgot about a while ago! 🙈 We don't have the framework linked at all for `BraintreePayPalNativeCheckout`.